### PR TITLE
image: fix dependencies at dev versions

### LINKED ISSUE #544: Join Node Fail
After the first cluster node being install, requesting the second node to join failed 

1) setup node one and let the dashboard running and copy the key. 
2) setup second node and join first node with the key 
3) in glass installer step 4 join fail . 
```

[INFO ] 2021-06-07T06:42:25 -- h11_impl -- 192.168.128.2:33804 - "PUT /api/orch/hostname HTTP/1.1" 200
[INFO ] 2021-06-07T06:42:25 -- h11_impl -- 192.168.128.2:33804 - "PUT /api/orch/hostname HTTP/1.1" 200
[DEBUG] 2021-06-07T06:42:25 -- nodes -- api > join 192.168.128.101:1337 with 3dbc-818d-66f3-47ab
[DEBUG] 2021-06-07T06:42:25 -- mgr -- join > with leader 192.168.128.101:1337, token: 3dbc-818d-66f3-47ab
[DEBUG] 2021-06-07T06:42:25 -- deployment -- join > with leader 192.168.128.101:1337, token: 3dbc-818d-66f3-47ab
[INFO ] 2021-06-07T06:42:25 -- h11_impl -- 192.168.128.2:33806 - "GET /api/local/status HTTP/1.1" 200
[INFO ] 2021-06-07T06:42:25 -- h11_impl -- 192.168.128.2:33806 - "GET /api/local/status HTTP/1.1" 200
[DEBUG] 2021-06-07T06:42:25 -- protocol -- client - state = CONNECTING
[DEBUG] 2021-06-07T06:42:25 -- protocol -- client - event = connection_made(<_SelectorSocketTransport fd=8 read=idle write=<idle, bufsize=0>>)
[DEBUG] 2021-06-07T06:42:25 -- client -- client > GET /api/nodes/ws HTTP/1.1
[DEBUG] 2021-06-07T06:42:25 -- client -- client > Headers([('Host', '192.168.128.101:1337'), ('Upgrade', 'websocket'), ('Connection', 'Upgrade'), ('Sec-WebSocket-Key', '7ypfDGaH1n77vn8Egz6b2w=='), ('Sec-WebSocket-Version', '13'), ('Sec-WebSocket-Extensions', 'permessage-deflate; client_max_window_bits'), ('User-Agent', 'Python/3.8 websockets/8.1')])
[DEBUG] 2021-06-07T06:42:25 -- protocol -- client - event = data_received(<251 bytes>)
[DEBUG] 2021-06-07T06:42:25 -- client -- client < HTTP/1.1 500 Internal Server Error
[DEBUG] 2021-06-07T06:42:25 -- client -- client < Headers([('Date', 'Mon, 07 Jun 2021 06:42:26 GMT'), ('Server', 'Python/3.8 websockets/8.1'), ('Content-Length', '76'), ('Content-Type', 'text/plain'), ('Connection', 'close')])
[DEBUG] 2021-06-07T06:42:25 -- protocol -- client ! failing CONNECTING WebSocket connection with code 1006
[DEBUG] 2021-06-07T06:42:25 -- protocol -- client x half-closing TCP connection
[DEBUG] 2021-06-07T06:42:25 -- protocol -- client - event = eof_received()
[DEBUG] 2021-06-07T06:42:25 -- protocol -- client - event = connection_lost(None)
[DEBUG] 2021-06-07T06:42:25 -- protocol -- client - state = CLOSED
[DEBUG] 2021-06-07T06:42:25 -- protocol -- client x code = 1006, reason = [no reason]

```

### PR DESCRIPTION
**Pull request by jecluis**
Otherwise we might end up with mismatches in behavior as pip packages
get updated.

Resolves #544

Signed-off-by: Joao Eduardo Luis \<joao@suse.com>

### CODE REVIEW

**Comment by jecluis** [comment]
Can't tell why tox is failing here, but it's doing so installing dependencies, and it's not reproducible locally. Also, nothing on this PR changes dependencies used for tox. This seems a github thing more than a PR thing.

**Comment by kshtsk** [comment]
jenkins test ubuntu

**Review by asettle — APPROVED** [review]

**Comment by jecluis** [comment]
ubuntu run failing due to missing kiwi package during installation:

```
14:24:04 >>> Package python3-kiwi is not available, but is referred to by another package.
14:24:04 >>> This may mean that the package is missing, has been obsoleted, or
14:24:04 >>> is only available from another source
14:24:04 >>> 
14:24:04 EEE E: Package 'python3-kiwi' has no installation candidate
14:24:04 EEE E: Unable to locate package python3-kiwi-boxed-plugin
14:24:04 >>> Dependency installation failed
```

Ignoring it as it has nothing to do with this particular change.

File: images/aquarium/config.sh [hunk 1]
```diff
@@ -105,7 +105,7 @@ if [[ "$kiwi_profiles" == *"Vagrant"* ]]; then
 	chown -R vagrant:vagrant /vagrant
 fi
 
-pip install fastapi uvicorn \
+pip install fastapi==0.63.0 uvicorn==0.13.3 websockets==8.1 \
```
Comment (mgfritch) [inline images/aquarium/config.sh:108]: should we use `requirements.txt` here?

Comment (jecluis) [reply images/aquarium/config.sh:108]: we would have to package `requirements.txt`, and it's nonetheless preferable to rely on the distro provided packages than `pip` as a general rule.

**Pull request merged**